src/cross_section_measurement/make_control_plots_fromTrees.py

```diff
@@ -39,7 +39,7 @@ def getHistograms( histogram_files,
     # Names of QCD regions to use
     qcd_data_region = ''
     qcd_data_region_electron = 'QCDConversions'
-    qcd_data_region_muon = 'QCD non iso mu+jets 3toInf'
+    qcd_data_region_muon = 'QCD non iso mu+jets 1p5to3'
```

Comment:
Does this mean the purposes of the two control regions have swapped?  Or is a temporary thing while the muon control region is being debugged?  Ultimately, it doesn't really matter which control region we use for the main result and for the systematic (though we should be able to justify it).